### PR TITLE
Add type check for json address import

### PR DESCRIPTION
**Pull request by joelamouche**
- when importing json file, if eth parachain, check that pair type is ethereum

### CODE REVIEW

File: packages/page-accounts/src/modals/Import.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import type { KeyringPair, KeyringPair$Json } from '@polkadot/keyring/types';
 import type { ActionStatus } from '@polkadot/react-components/Status/types';
 import type { ModalProps } from '../types';
 
-import React, { useCallback, useMemo, useState } from 'react';
+import React, { Dispatch, SetStateAction, useCallback, useMemo, useState } from 'react';
```
Comment (jacogr) [inline packages/page-accounts/src/modals/Import.tsx:8]:
```suggestion
import React, { useCallback, useMemo, useState } from 'react';
```

And then do an explict type-only import at the top -

```js
import type { Dispatch, SetStateAction } from 'react';
```

**Review by jacogr — COMMENTED** [review]
One tiny comment. Not sure why CI is not running on this one :(

**Comment by joelamouche** [comment]
@jacogr updated and ready

File: packages/apps-config/package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "@sora-substrate/type-definitions": "1.3.34",
     "@subsocial/types": "0.5.9-dev.2",
     "@zeitgeistpm/type-defs": "0.2.0",
-    "moonbeam-types-bundle": "1.1.25",
+    "moonbeam-types-bundle": "latest",
```
Comment (jacogr) [inline packages/apps-config/package.json:28]: We actually want this on a specific version - we had some issues with people using the `@polkadot/apps-config` package (in other projects) running into issues here.

I know it is not perfect, but at the same time it does open up gaps.

Comment (jacogr) [inline packages/apps-config/package.json:28]:
```suggestion
    "moonbeam-types-bundle": "1.2.26",
```

**Review by jacogr — APPROVED** [review]
Thank you!

**Review by jacogr — COMMENTED** [review]
One small conflict, cannot fix it myself.

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**